### ISSUE TITLE
Add a Gridfinity example

### ISSUE THREAD

**Issue by bschwind** [bschwind/opencascade-rs#102]
[Reference](https://gridfinity.xyz/)

This would serve as a great base part to both show off the API, and act as a base for any parts you'd want to make which integrate with gridfinity.

**Comment by zignig** [comment]
going to have a go at this one.

**Comment by zignig** [comment]
ref https://gridfinity.xyz/specification/

**Comment by bschwind** [comment]
@zignig please do!

**Comment by zignig** [comment]
it's a start.
![plate](https://github.com/bschwind/opencascade-rs/assets/109033/a3e8cacb-86d9-4e6e-8119-f04a69886d33)

**Comment by bschwind** [comment]
@zignig  looking great! Did you export to blender and raytrace it?

**Comment by zignig** [comment]
Yes, blender is great. I need to port part of the cqparts code base across so I can automagically generate blender renders. 

FYI , the code so far is here. https://github.com/zignig/cascade_rs_investigations/tree/main/gridfin , I will write up my findings in the discussions, as there are some observations I have about the ergonometrics of the rust interface.

**Comment by zignig** [comment]
mmm nice. 
![gridfin](https://github.com/bschwind/opencascade-rs/assets/109033/20189e6e-6dcb-4b5e-92ce-d3798a874d1b)

**Comment by bschwind** [comment]
@zignig very nice!!

You're making me want auto-exports to blender even more now.

Combined with [rendering from the CLI](https://docs.blender.org/manual/en/latest/advanced/command_line/render.html), we could have CI generate nice looking renders for pull requests. That would be cool

**Comment by zignig** [comment]
https://github.com/cqparts/cqparts_webapi/blob/master/blender/base_test.py
